### PR TITLE
Fine tunned code

### PR DESCRIPTION
**Pull request by calistus**
All text in the layout are now in the String file

### CODE REVIEW

File: app/src/main/res/values/strings.xml [hunk 1]
```diff
@@ -0,0 +1,10 @@
+<resources>
+    <string name="app_name">PCSA</string>
+    <string name="hello_world">Hello world!</string>
```
Comment (sandarumk) [inline app/src/main/res/values/strings.xml]: Is the "Hello World" string really necessary?

File: app/src/main/res/layout/fragment_main.xml [hunk 2]
```diff
@@ -0,0 +1,76 @@
...
+        android:orientation="vertical"
+        >
+        <TextView
+            android:layout_width="fill_parent"
+            android:layout_height="wrap_content"
+            android:gravity="center"
+            android:textStyle="bold|italic"
+            android:textSize="20sp"
+            android:textColor="#fffaf3f1"
+            android:text="A confidential Safety resource for Peace Corps Volunteers"
```
Comment (sandarumk) [inline app/src/main/res/layout/fragment_main.xml]: Move the string to strings.xml file

**Comment by rosariorobinson** [comment]
Where are we on this PR? I need to merge to Peace corps and there are some conflicts.

**Comment by sandarumk** [comment]
This PR is not ready to be merged. Ilo has to do the suggested changes and do another PR.

**Comment by calistus** [comment]
I sent a PR before with the suggested changes but don't know why the new
code didn't show up but the message was on Github
I just sent a new PR now also. Check it out at:
https://github.com/systers/PCSA/pull/6
but before sending the PR, I had the attached error on my git panel.

**Comment by calistus** [comment]
I sent a PR before with the suggested changes but don't know why the new code didn't show up but the message was on Github
I just sent a new PR now also. Check it out at: https://github.com/systers/PCSA/pull/6
but before sending the PR, I had the attached error on my git panel.
error message

**Comment by calistus** [comment]
I just saw some instructions on how I can fix this on github.
I performed the suggested command, that is:

git checkout -b calistus-master develop

but I am getting a warning that some files may be over written.
So Dinu I would require your expertise to fix this.
Would you be available tomorrow on Hangout so that I can share my screen
while we fix it together?

**Comment by sandarumk** [comment]
Ilo let me be clear. Have you committed all changes to the app you developed in your forked repository? It just shows me one commit from you which is the one the PR is initially based on. It doesn't show you any new changes which should be there. So first make your forked repository updated and let me know. I assume that you haven't pushed your work to the fork repository and still they are on your working directory or staging index (that means committed, not pushed to remote) .

**Comment by calistus** [comment]
I just sent a PR to the forked repo.
Here is the link: https://github.com/calistus/PCSA/pull/1
Please can you still check if you can merge that.

**Pull request merged**